fdbmonitor/CMakeLists.txt

```diff
@@ -4,11 +4,12 @@ add_executable(fdbmonitor ${FDBMONITOR_SRCS})
 strip_debug_symbols(fdbmonitor)
 assert_no_version_h(fdbmonitor)
 if(UNIX AND NOT APPLE)
-    target_link_libraries(fdbmonitor rt)
+    target_link_libraries(fdbmonitor PUBLIC rt)
```

Comment:
Why PUBLIC? I don't think anything links agains fdbmonitor since it's an executable

Reply:
Because I don't really understand CMake PRIVATE vs PUBLIC and this seemed like the safer option 😛 

If you tell me that PRIVATE makes more sense here, then I can do that.

Reply:
As far as I can tell there's no reason to prefer one over the other, but we seem to use PRIVATE elsewhere for linking things to executables, so I guess I prefer to follow that convention.

Reply:
```suggestion
    target_link_libraries(fdbmonitor PRIVATE rt)
```